etc/ci_scripts/dump_artifacts/nomad_artifacts.py

```diff
@@ -69,7 +69,7 @@ def _dump_hashicorp_agent_logs(artifacts_dir: Path) -> None:
 class NomadNamespace:
     name: str
 
-    def __init__(self, input: Dict[str, Any]) -> None:
+    def __init__(self, input: dict[str, Any]) -> None:
```

Comment:
Same here, I've always assumed that `List` and `Dict` were preferred to `list` and `dict` but I could absolutely be wrong